Firestore/core/src/remote/bloom_filter.h

```diff
@@ -63,6 +82,14 @@ class BloomFilter final {
 
   // Bloom filter's bitmap.
   std::vector<uint8_t> bitmap_;
+
+  BloomFilter::Hash Md5HashDigest(const absl::string_view key) const;
```

Comment:
Please remove the `BloomFilter::` prefix on `BloomFilter::Hash` wherever possible, here and elsewhere in this file.

Reply:
While `BloomFilter::` removed, it gives error message: 
![Screenshot 2023-02-22 at 10 06 34 PM](https://user-images.githubusercontent.com/107142260/220832257-24d3ec8d-11e7-4ad7-b944-244c7ffc0b6d.png)

Reply:
That's fine. If the compiler requires it, then put it in. But I think there are some places in your code where it can be omitted. Please do a quick check of all occurrences of `BloomFilter::Hash` and see which ones can be replaced by simply `Hash`.